src/scipp/configuration/config_default.yaml

```diff
@@ -26,109 +26,3 @@ colors:
 
 # Maximum number of rows in a table to display at the same time.
 table_max_size: 50
-
-# Default style of plots produced by scipp.
```

Comment:
Is this found in Plopp, or are the settings no longer required?

Reply:
There is no config in Plopp currently.
We may add one again in the future, if users want one.